apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,19 @@
 # API Changelog
 
+## BFD-1519: Map Revenue Center Unit Count in V2
+
+This extension was not previously available in the v2 claims for revenue center unit count. The
+extension is now available in the extensions for Inpatient, Outpatient, HHA, Hospice, and SNF claims.
```

Comment:
I would change this last sentence to say: "The following claim types will have this extension available: Inpatient, Outpatient, HHA, Hospice, and SNF."

Reply:
done